src/simulations/local_elec.py

```diff
@@ -555,14 +558,15 @@ def policy_to_rewardvar(
         else:
             netp0 = []
         if self.prm['grd']['manage_voltage']:
-            # retrieve info from battery
-            self.car.active_reactive_power_car()
-            # q_car_flex will be a decision variable
-            # p_car_flex is needed to set apparent power limits
-            p_car_flex = self.car.p_car_flex
-            q_car_flex = self.car.q_car_flex
+            if not self.reactive_power_for_voltage_control:
+                # retrieve info from battery if not a decision variable
+                self.car.active_reactive_power_car()
```

Comment:
maybe change the method name to reflect it outputs the reactive power _if_ the power factor is fixed?